src/firebase/firebase_time_format_converter.cr

```diff
@@ -0,0 +1,13 @@
+module Google
+  class FirebaseAuth
+    module TimeFormatConverter
+      def self.from_json(value : JSON::PullParser) : Time
+        Time::Format::RFC_3339.parse(value.read_string)
+      end
+
+      def self.to_json(value : Time, json : JSON::Builder) : Nil
+        json.string(Time::Format::RFC_3339.format(value))
+      end
+    end
+  end
+end
```

Comment:
Noticed this https://github.com/PlaceOS/google/blob/master/src/calendar/events.cr#L11
Might be able to use that instead.

It might be a bit cleaner to put these in their own folder outside of the firebase scope given they're likely to be used for other APIs